once/docker-entrypoint.sh

```diff
@@ -424,6 +426,11 @@ docker_authgear_first_time_setup() {
 		--smtp-sender-address="$AUTHGEAR_SMTP_SENDER_ADDRESS" \
 		--search-implementation=postgresql \
 		-o "$init_output_accounts"
+
+	# Disable public signup
+	yq <"$init_output_accounts/authgear.yaml" --yaml-output '.authentication.public_signup_disabled = true' >"$init_output_accounts/authgear.yaml.tmp"
```

Comment:
Can we do this in authgear init?

Reply:
Actually yes, let me add a new question and a new flag in `init`.

Reply:
Added 2 commits to address this comment.